js/test/test.js

```diff
@@ -141,25 +163,37 @@ async function test (methodName, exchange, ... args) {
     }
 }
 
-async function testSymbol (exchange, symbol) {
+async function testSafe(methodName, exchange, ...args) {
+    try {
+        if (exchange[methodName] === undefined || tests[methodName] === undefined) {
+            return true; // skip if not implemented
+        }
+        await test(methodName, exchange, ...args);
+        return true;
+    } catch (e) {
+        return false;
```

Comment:
what about this ? 
```suggestion
        return '';
    } catch (e) {
        return exceptionMessage(e);
```

and down below altering to this:
```
        const errorMessage = await testSafe (testName, ...testArgs);
        if (errorMessage !== '') {
            errors.push (testName + ' : ' + errorMessage );
        }
```

Reply:
@ttodua we're already logging it inside `test` so I don't think we should return it again because let's say 10 tests failed, it would be a mess to throw all of that inside a single exception so I think we can keep it simple here

Reply:
yes, I see. ok, good.